src/main/webapp/js/community-page-loader.js

```diff
@@ -1,33 +1,57 @@
+const React = window.React;
+const ReactDOM = window.ReactDOM;
+const h = React.createElement;
+const root = document.getElementById("list");
```

Comment:
The root element should generally be a div; React will insert its rendered HTML into the root. Since you render an `li` in your React, this code will produce two nested `li`s, which is not what you want.